wsdlgo/encoder.go

```diff
@@ -280,13 +280,25 @@ func (ge *goEncoder) importRemote(url string, v interface{}) error {
 		return nil
 	}
 
-	resp, err := ge.http.Get(url)
-	if err != nil {
-		return err
+	var r io.Reader
+	if strings.HasPrefix(url, "http") {
```

Comment:
This wouldn't allow local files called `http*`. We should probably use `url.Parse()` here and look for the presence of a scheme in the returned [url](https://golang.org/pkg/net/url/#URL). It will parse file names correctly, with an empty `.Scheme` but correct `.Path`.